test/cross-browser/report.js

```diff
@@ -270,8 +308,10 @@ describe('RcSocketIntegration', function () {
             .then(waitASecond)
             .then(stopServerSocket)
             .then(waitASecond)
+            .then(waitASecond)
             .then(startServerSocket)
             .then(waitASecond)
+            .then(waitASecond)
```

Comment:
`waitTwoSeconds`? 😂 Haha sorry, I am just realizing how gritty and hacky this project was.

To cleaner and less hacky code in the new year 🍻